code/datums/components/butchering.dm

```diff
@@ -138,7 +140,22 @@
 /datum/component/butchering/recycler
 
 
-/datum/component/butchering/recycler/Initialize(_speed, _effectiveness, _bonus_modifier, _butcher_sound, disabled, _can_be_blunt)
+/datum/component/butchering/recycler/Initialize(
+	speed,
+	effectiveness,
+	bonus_modifier,
+	butcher_sound,
+	disabled,
+	can_be_blunt,
+)
+	src.speed = speed
```

Comment:
I didn't mean this, I meant ``butcher_callback`` is missing in the args

Reply:
ah sorry

Reply:
done